lib/Repositories/ExpoFileDriver.php

```diff
@@ -90,6 +87,9 @@ public function forget(string $key, string $value = null): bool
             return false;
         }
 
+        // @todo BUG count($storageInstance->{$key}) > 1 should be > 0, because we never check if $value is
```

Comment:
I don't get this bug could you explain it more? also we can write a test scenario to prove that there's a bug here ;)